builder/virtualbox/common/driver_mock.go

```diff
@@ -81,6 +84,11 @@ func (d *DriverMock) CreateNVMeController(vm string, controller string, portcoun
 	return d.CreateNVMeControllerErr
 }
 
+func (d *DriverMock) RemoveFloppyControllers(vm string) error {
```

Comment:
It would be nice to validate if the RemoveFloppyControllers is being used correctly in one of the existing tests in `builder/virtualbox/common/step_attach_floppy_test.go` using this mock.

Reply:
Updated with a simple test